### PR TITLE
`trace list -t mf` - now can use external dictionary keys file

### PR DESCRIPTION
**Pull request by McEloff**

### CODE REVIEW

File: client/src/cmdtrace.c [hunk 1]
```diff
@@ -626,6 +630,7 @@ int CmdTraceList(const char *Cmd) {
         arg_lit0("x", NULL, "show hexdump to convert to pcap(ng)\n"
                  "                                   or to import into Wireshark using encapsulation type \"ISO 14443\""),
         arg_strx0("t", "type", NULL, "protocol to annotate the trace"),
+        arg_strx0("d", "dict", NULL, "use dictionary keys file"),
```
Comment (iceman1001) [inline client/src/cmdtrace.c:633]: we tend to use -d for data..
I suggest you replace it with  NULL,  "dict"   making a long param only.   Don't forget to change the command sample text above too.

**Comment by iceman1001** [comment]
Awesome contribution!   Just minor thing and I can merge it

File: client/src/cmdtrace.c (hunk 1 above)
Comment (McEloff) [reply client/src/cmdtrace.c:633]: You are right. But "f" flag for file selection is used now for "show frame delay times".

Comment (iceman1001) [reply client/src/cmdtrace.c:633]: yeah,  we are converting from old style to cliparser style and in that process we will have do go over stuff a second and third time to iron out those quirks.   So don't take mixes that still in the client as a justification.
We are trying to make the conversion work less.

**Comment by iceman1001** [comment]
Awesome!

**Pull request merged**